modular-docs-manual/content/topics/module_guidelines-procedure.adoc

```diff
@@ -32,12 +32,14 @@ The procedure consists of one or more steps required to complete the procedure.
 
 For single-step procedures, use an unnumbered bullet instead of a numbered list.
 
+NOTE: Not all numbered lists in documentation are procedures. You can also use numbered lists in any module type for non-procedural sequences, such as a process flow of system actions.
+
 [discrete]
-== Procedure verification steps
+== Procedure Verification Steps
```

Comment:
I changed to headline style against prescribed convention of sentence case strictly for consistency because the entire manual is headline style, out of compliance. Issue #133 will at some point address the headline style issue throughout. Several other mechanical issues need to be addressed throughout also, but that will be done as a separate effort.